packages/rax-use-fetch/src/index.js

```diff
@@ -0,0 +1,13 @@
+import { useMemo } from 'rax';
+import usePromise from 'rax-use-promise';
+
+export default function useFetch(input, init, responseType = 'json') {
+  return usePromise(
+    useMemo(
```

Comment:
use memo should be used in performance optim, not a universal logic.    
Consider that API returns current time or a real-time number of sales.